apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -45,12 +46,17 @@ export default class StartNewProject extends React.Component {
   };
 
   render() {
-    const { canViewAdvancedTools, canViewFullList, includeDanceParty } = this.props;
+    const { canViewAdvancedTools, canViewFullList } = this.props;
+    const { includeDanceParty, includeMCAquatic } = this.props;
     const { showFullList } = this.state;
     const GAMES_AND_EVENTS =  includeDanceParty ?
       ['spritelab', 'dance','flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball'] :
       ['spritelab','flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball'];
 
+    const MINECRAFT = includeMCAquatic ?
```

Comment:
nit: you can use the `unshift` method to prepend to an array and avoid repetition:

```
let minecraft_buttons = ['minecraft_hero', 'minecraft_designer', 'minecraft_adventurer'];
if (includeMCAquatic) {
  minecraft_buttons.unshift('minecraft_aquatic');
}
```